Formula/cdo.rb

```diff
@@ -0,0 +1,35 @@
+class Cdo < Formula
+  desc "Climate Data Operators"
+  homepage "https://code.mpimet.mpg.de/projects/cdo"
+  url "https://code.mpimet.mpg.de/attachments/download/20826/cdo-1.9.8.tar.gz"
+  sha256 "f2660ac6f8bf3fa071cf2a3a196b3ec75ad007deb3a782455e80f28680c5252a"
+
+  depends_on "eccodes"
+  depends_on "hdf5"
+  depends_on "netcdf"
+  depends_on "proj"
+  depends_on "szip"
+
+  def install
+    args = %W[
+      --disable-dependency-tracking
+      --prefix=#{prefix}
+      --with-eccodes=#{Formula["eccodes"].opt_prefix}
+      --with-netcdf=#{Formula["netcdf"].opt_prefix}
+      --with-hdf5=#{Formula["hdf5"].opt_prefix}
+      --with-szlib=#{Formula["szip"].opt_prefix}
+    ]
+
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  test do
+    data = <<~EOF.unpack1("m")
+      R1JJQgABvAEAABz/AAD/gAEBAABkAAAAAAEAAAoAAAAAAAAAAAAgAP8AABIACgB+9IBrbIABLrwA4JwTiBOIQAAAAAAAAXQIgAPEFI2rEBm9AACVLSuNtwvRALldqDul2GV1pw1CbXsdub2q9a/17Yi9o11DE0UFWwRjqsvH80wgS82o3UJ9rkitLcPgxJDVaO9No4XV6EWNPeUSSC7txHi7/aglVaO5uKKtwr2slV5DYejEoKOwpdirLXPIGUAWCya7ntil1amLu4PCtafNp5OpPafFqVWmxaQto72sMzGQJeUxcJkbqEWnOKM9pTOlTafdqPCoc6tAq0WqFarTq2i5M1NdRq2AHWzFpFWj1aJtmAOrhaJzox2nwKr4qQWofaggqz2rkHcog2htuI2YmOB9hZDIpxXA3ahdpzOnDarjqj2k0KlIqM2oyJsjjpODmGu1YtU6WHmNZ5uljcbVrduuOK1DrDWjGKM4pQCmfdVFprWbnVd7Vw1QY1s9VnNzvZiLmGucPZwVnM2bm5yFqb2cHdRQqs2hhZrrm1VGeEQgOduhjbWrqAWfzaANnZOdWJ0NnMWeJQA3Nzc3AAAAAA==
+    EOF
```

Comment:
If this data is also available online it might be easiest to download it with a resource block

Reply:
I mentioned already that the test case is only available as part of the source package. There is no way to download individual files from the upstream, only the complete package. I just picked one of the smaller files for testing basic functionality.

Reply:
Doesn't have to be data from the test case. Maybe there was some other data available online that the program could work with.